pkgs/development/python-modules/mkdocs-git-revision-date-localized-plugin/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, gitpython
+, mkdocs
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-git-revision-date-localized-plugin";
+  version = "1.2.0";
```

```suggestion
  version = "1.2.0";
  format = "setuptools";
```
Would you please add format explicitly?

---

Done!